pkg/kapis/devops/v1alpha2/devops.go

```diff
@@ -440,6 +410,9 @@ func (h *ProjectPipelineHandler) hasSubmitPermission(req *restful.Request) (hasP
 		err = errors.New("cannot get the submitters of current pipeline run")
 		return
 	}
+	if !hasPermit && err == nil {
+		err = fmt.Errorf("you have no permission to approve this pipeline run")
```

Comment:
`pipeline run` or `pipelinerun`, or `pipeline run record`

We don't have a expression `pipeline run`